lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
+  @moduledoc false
+
+  alias Screens.V2.WidgetInstance.ReconstructedAlert
+
+  defstruct alert: nil
+
+  @type t :: %__MODULE__{
+          alert: String.t()
```

Comment:
This type should be `Alert.t()`.

Reply:
Surprising that this compiled properly...